src/index.html

```diff
@@ -13,6 +13,76 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <section class="stars--0">
+      <article class="stars__star"></article>
+
+      <article class="stars__star"></article>
+
+      <article class="stars__star"></article>
+
+      <article class="stars__star"></article>
+
+      <article class="stars__star"></article>
+    </section>
+
+    <section class="stars--1">
+      <article class="stars--1__star"></article>
+
+      <article class="stars__star"></article>
+
+      <article class="stars__star"></article>
+
+      <article class="stars__star"></article>
+
+      <article class="stars__star"></article>
+    </section>
+
+    <section class="stars--2">
+      <article class="stars--2__star"></article>
+
+      <article class="stars--2__star"></article>
+
+      <article class="stars__star"></article>
+
+      <article class="stars__star"></article>
+
+      <article class="stars__star"></article>
+    </section>
+
+    <section class="stars--3">
+      <article class="stars--3__star"></article>
+
+      <article class="stars--3__star"></article>
+
+      <article class="stars--3__star"></article>
+
+      <article class="stars__star"></article>
+
+      <article class="stars__star"></article>
+    </section>
+
+    <section class="stars--4">
```

Comment:
The class name `stars--4` does not represent the meaning of the content. Use class names that represent the meaning of the content, not the styles or tag names. Consider renaming this class to something more meaningful.